### ISSUE TITLE
Allow customizing keyboard shortcuts

### ISSUE THREAD

**Issue by Varriount** [WinMerge/winmerge#1477]
Allow configuring keyboard shortcuts associated with various actions.

**Comment by TFWol** [comment]
...I can't believe this isn't a thing with this type of program.

**Comment by Varriount** [comment]
> ...I can't believe this isn't a thing with this type of program.

Do you know how to implement this?

**Comment by TFWol** [comment]
> > ...I can't believe this isn't a thing with this type of program.
> 
> Do you know how to implement this?

Nope, just expressing the same sentiment.

Lately I've been using [Lintalist](https://github.com/lintalist/lintalist) to work around these annoyances.

**Comment by jesus2099** [comment]
Oh yes, I was once a heavy user of [AutoHotkey](https://www.autohotkey.com).
Maybe that's a reason to use it again…

**Comment by TFWol** [comment]
@jesus2099 Yeah, I really like ahk since I can review the code fairly easily. 
I did go overboard and made a bunch of hotstrings and keyboard shortcuts, which was getting out of hand, until I stumbled across that Lintalist tool.

I wish I knew about that repo ages ago.

**Comment by lededev** [comment]
I need both shortcut and shortcut to custom scripts, because I want to use shortcut keys to copy the current line, for example, Ctrl+1 copies the current line in the left window, Ctrl+2 copies the current line in the middle window, and Ctrl+3 copies the current line in the right window.

**Comment by lededev** [comment]
I finally implemented the function I wanted by myself, using hotkeys Ctrl+1 to 3 to copy the currently selected difference to the system clipboard. The code is [here](https://github.com/lededev/winmerge/commit/57cc02cf43fd755c646b277c54856c222525a168). If you need custom hotkeys, you can refer to how to add your own hotkeys.
